bgpd/bgp_route.c

```diff
@@ -8208,6 +8208,15 @@ DEFUN (show_ip_bgp,
   return bgp_show (vty, NULL, AFI_IP, SAFI_UNICAST, bgp_show_type_normal, NULL, use_json(argc, argv));
 }
 
+ALIAS (show_ip_bgp,
+       show_bgp_ipv4_cmd,
+       "show bgp ipv4 {json}",
```

Comment:
I'm ok with this change in retrospect, it is merely calling an existing function to allow another way to get the same info